packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -26,48 +27,69 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface {
 
     bytes32 identifier = "ASSERT_TRUTH";
 
-    constructor(address _finderAddress) {
-        finder = FinderInterface(_finderAddress);
+    IERC20 public defaultCurrency;
```

Comment:
I did: a) moved this file to a different directory b) add default props c) added a function to set these defaults.